templates/profile.html.twig

```diff
@@ -0,0 +1,40 @@
+{#
+ # -------------------------------------------------------------------------
+ # News plugin for GLPI
+ # -------------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of News.
+ #
+ # News is free software; you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation; either version 2 of the License, or
+ # (at your option) any later version.
+ #
+ # News is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with News. If not, see <http://www.gnu.org/licenses/>.
+ # -------------------------------------------------------------------------
+ # @copyright Copyright (C) 2015-2022 by News plugin team.
+ # @license   GPLv2 https://www.gnu.org/licenses/gpl-2.0.html
+ # @link      https://github.com/pluginsGLPI/news
+ # -------------------------------------------------------------------------
+ #}
+
+<div class="spaced">
+    <form method="post" action="{{ 'Profile'|itemtype_form_path }}">
+        {% do profile.displayRightsChoiceMatrix(rights, {'title': title}) %}
+
+        <div class="center">
+            <input type="hidden" name="id" value="{{ id }}">
+            {{ call('Html::submit', [_x('button', 'Save'), {'name': 'update'}])|raw }}
```

Comment:
`call()` should be used only when there is no other option. Here you could probably write the HTML code of button directly.

Reply:
I'm conflicted about this one, the submit button should be the same everywhere and not calling `Html::submit` would mean duplicating some code that must be maintained individually (as `Html::submit()` seems to contain some logic that must be shared for all buttons).

GLPI core should offer a twig macro or template for the submit button to avoid using `call` here.

Reply:
We can keep it for now, but, indeed, we should add the submit button to GLPI core macros.